src/main/java/discord4j/discordjson/json/GuildScheduledEventModifyRequest.java

```diff
@@ -47,4 +47,7 @@ static ImmutableGuildScheduledEventModifyRequest.Builder builder() {
     Possible<Integer> status();
 
     Possible<String> image();
+
+    @JsonProperty("recurrence_rule")
+    Possible<RecurrenceRuleData> recurrenceRule();
```

Comment:
If https://github.com/discord/discord-api-docs/pull/7062 is merged then this need to be change to
```suggestion
    Possible<Optional<RecurrenceRuleData>> recurrenceRule();
```